### DISCUSSION TITLE
Drag&Drop events

### DISCUSSION THREAD

**Discussion by 4www44** [Etar-Group/Etar-Calendar#1289] [Ideas]
It would be very nice to add Google calendar like feature: when you long press on any event it selects it and then without rising finget you move it to new time or even another day. And when you rise finget the event "lands" to new day/time.
This feature will be way more quick than to open an event, then change date and time, then click "Save". Especially for those who move thier event very often.

Here's an example - https://support.google.com/calendar/answer/72143?hl=en&co=GENIE.Platform%3DAndroid#zippy=%2Cedit-an-event - Edit or cancel events -> "Tip: To move an event, **drag and drop it to a new time and date** on your calendar."

**Comment by dexic79** [comment]
I would like to have that, too. It would ease up busy days as a road warrior a lot!

**Comment by mohammadfarari1360** [comment]
Ok

**Comment by xtofr** [comment]
Plus 1
😊
